controllers/apply/form-router-next/steps.js

```diff
@@ -237,12 +237,24 @@ module.exports = function(formId, formBuilder) {
                         }
                     }
 
-                    const { nextPage } = form.pagination({
-                        baseUrl: res.locals.formBaseUrl,
-                        sectionSlug: req.params.section,
-                        currentStepIndex: stepIndex,
-                        copy: res.locals.copy
-                    });
+                    let redirectUrl;
+                    if (req.body.previousBtn) {
+                        const { previousPage } = form.pagination({
```

Comment:
We don't need to call this twice, you can get these values from the same call. Also worth extracting into a named function like

```js
function determineRedirectUrl() {
    const { previousPage, nextPage } = form.pagination({
        baseUrl: res.locals.formBaseUrl,
        sectionSlug: req.params.section,
        currentStepIndex: stepIndex,
        copy: res.locals.copy
    });

    return req.body.previousBtn
        ? previousPage.url
        : nextPage.url;
}
```